presto-product-tests/conf/docker/singlenode-ldap-referrals/presto-master/etc/ldap-authenticator-referrals.properties

```diff
@@ -0,0 +1,5 @@
+password-authenticator.name=ldap
```

Comment:
Rename this file to `password-authenticator.properties`. That way it will be more clear what it does.